sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/bigquery/BigQueryServicesImplTest.java

```diff
@@ -498,7 +498,7 @@ public void testInsertRetry() throws Exception {
 
```

Comment:
I think change the insertAll() to the following will help testing and support additional configurations. (the handling of ignoreUnknownValues will then stay in BigQueryIO.)
void insertAll(TableReference ref, Collection<TableDataInsertAllRequest> request);

https://issues.apache.org/jira/browse/BEAM-1306